src/fat_beagle.hpp

```diff
@@ -44,42 +45,66 @@ class FatBeagle {
   // This override performs a "classical" log likelihood calculation of a rooted tree
   // considered as an unrooted tree with no time-tree extras.
   double UnrootedLogLikelihood(const RootedTree &tree) const;
-  double LogLikelihood(const RootedTree &tree) const;
+  double LogLikelihood(const RootedTree &tree,
+                       const std::optional<PhyloFlags> flags = std::nullopt) const;
```

Comment:
const in unnecessary here for std::optional argument

Reply:
@ognian- I'd like to know why this is. We can discuss tomorrow?